### DISCUSSION TITLE
Feature Request: track "authentication requests"

### DISCUSSION THREAD

**Discussion by wars2k** [CyferShepard/Jellystat#20] [General]
I would love to be able to track login attempts to my Jellyfin server to see how many attempted logins are being made as well as possibly the IP address they come from, etc. I'm not sure if this is within the scope of the project, but I think it could be a useful metric. 

This event occurs in the logs as follows: 

`[2023-04-08 13:01:18.508 -04:00] [INF] [109] Jellyfin.Server.Implementations.Users.UserManager: Authentication request for "admin" has succeeded.`

**Comment by CyferShepard** [comment]
Hi @wars2k . This seems really interesting, but I cant really see how this will used in a meaningfully way. It could be implemented but I'm not really sure if this is something that is widely requested or just a niche use case, plus I'm not sure how it would fit in with the other types of metrics, as its mostly Media and Playback Statistics, this seems like something in-between like Security Statistics?. Il keep this idea earmarked for now and if it gains more traction I may look into implementing it
